lib/hw_unique_key/hw_unique_key_kmu.c

```diff
@@ -11,22 +11,36 @@
 
 #define KMU_KEYSLOT_SIZE_WORDS 4
 
+// Use 'copy_of_uicr_word_read' instead of 'nrfx_nvmc_uicr_word_read'
+// until it has become available everywhere. Everywhere consists of
+// TF-M's copy of nrfx, nrfx, hal_nordic in Zephyr and NCS.
+
+static uint32_t copy_of_uicr_word_read(uint32_t volatile *address)
```

Comment:
Nit: This name could be more descriptive, e.g. uicr_word_read_with_dsb.

Reply:
nrfx_nvmc_uicr_word_read_with_dsb is too long of a function name IMHO.

I'd rather not put the details of how we workaround the  PAN in the function name. It might change in time.

Reply:
I'm also trying to be consistent with:

nrfx_nvmc_otp_halfword_read